package/src/components/Attachment/Card.tsx

```diff
@@ -116,11 +128,15 @@ const CardWithContext = <
     text,
     thumb_url,
     title,
+    type,
   } = props;
 
   const {
     theme: {
       colors: { accent_blue, black, blue_alice, transparent },
+      imageGallery: {
```

Comment:
This should come from Card. And the name can be `playButtonStyle` as Vish mentioned.